lib/fizzy/parser_expr.cpp

```diff
@@ -185,6 +185,20 @@ parser_result<Code> parse_expr(const uint8_t* pos)
             break;
         }
 
+        case Instr::br_table:
+        {
+            std::vector<uint32_t> label_indices;
+            std::tie(label_indices, pos) = parser<std::vector<uint32_t>>{}(pos);
+            uint32_t last_label_idx;
+            std::tie(last_label_idx, pos) = leb128u_decode<uint32_t>(pos);
+
+            push(code.immediates, label_indices.size());
```

Comment:
What is the type of this? Can you enforce an explicit type to be pushed as immediate?

Reply:
Yeah this probably works incorrectly without explicit cast.